platformio.ini

```diff
@@ -16,59 +16,53 @@ test_dir = firmware/test
 
 [env]
 platform = platformio/ststm32@^17.3.0
-
-# WARNING: For whatever reason, this CANNOT be the
-# disco_b_g431b_esc1 board you probably anticipate
-# it can be. This took my morning don't let it take yours.
 board = genericSTM32G431CB
 framework = arduino
 monitor_dtr = 1
 monitor_speed = 115200
-
-lib_deps =
-    # these both need to be on the dev branch, but we cannot spec a branch for deps
-    # instead we've taken the tip of the dev branches for each libs (as of 22-feb-2024)
-    https://github.com/simplefoc/Arduino-FOC-drivers.git#3a11d581d73018d3836614d674210e017da43661
-    https://github.com/simplefoc/Arduino-FOC.git#83e52478c14b6fffa8a547fe74f3b41e7bbfde0e
-
-build_flags =
-;    -flto
-    -DPIO_FRAMEWORK_ARDUINO_ENABLE_CDC
-    -DUSBCON
-    -DUSBD_VID=0x0483
-    -DUSBD_PID=0x5740
-    -DUSB_MANUFACTURER="ATO"
-    -DUSB_PRODUCT=""SPIN""
-    -DHAL_PCD_MODULE_ENABLED
-    -DHSE_VALUE=8000000
-    -DSIMPLEFOC_STM32_DEBUG
-    -DSPIN_SERVO_FIRMWARE_VERSION=\"1.0.0\"
-    -DHAL_CORDIC_MODULE_ENABLED
-
-monitor_filters =
-    send_on_enter
-    colorize
+; lib_deps = 
```

Comment:
Figure out how to re-compose these lib-deps into a "complete build" or something of the likes